doc/design.md

```diff
@@ -139,7 +139,7 @@ The function signature of `add` is:
 
 For more complicated expressions, instead of `add`, we might also consider
 using the naming scheme proposed in [1]. Another advantage of this approach is
-that compiler errors are much easier to understand, since it is either finding
+that compiler errors are much easier to understand since it is either finds
```

Comment:
since it either finds

Reply:
i have resolved the issue can you take a look @certik

Reply:
I have resolved the issue can you take a look @certik

Reply:
I have resolved the issue can you take a look @certik thanks for the suggestion :relaxed: